netlify.toml

```diff
@@ -42,7 +42,7 @@
 
 [context.deploy-preview.environment]
   VITE_SANITY_TAG = "default"
-  VITE_SANITY_DATASET = "staging"
+  VITE_SANITY_DATASET = "production"
```

Comment:
this is just for testing on the deploy preview since this is a hotfix and `main` doesn't have the sanity graphql API changes
will drop https://github.com/regen-network/regen-web/pull/2208/commits/9f1f83d93bea4cb2fbc3313080aff93e25082585 before merging